LukeMason33/Rotten-Tomatillos
# Feature sass single movie

### Is this a fix or a feature?

Feature.

### What is the change/fix?

This PR adds styling to the single movie page. Minor changes to the JSX in `single-movie-component` to setup error handling, and display dollar amounts as currency. It also adds a `/sass` folder which contains `normalize` `variables` and `element-styling` files.

### Where should the reviewer start?

Familiarize yourself with the `single-movie-component.scss` file, and the styling being applied.

### How should this be tested?

In browser, visit a single movie page and ensure that styling is being applied correctly.
